pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -270,6 +272,16 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests...")
+        self._slack_client.bind_channel(self.release_name)
```

Comment:
This will create another Slack thread besides the one created by Jenkinsfile. It would be nice to combine but that can be a separate PR.